sample/src/main/java/com/uber/rave/sample/DefaultSerializer.java

```diff
@@ -0,0 +1,53 @@
+package com.uber.rave.sample;
+
+import com.uber.rave.sample.github.model.Owner;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * This uses the default Java serializer.
+ */
+class DefaultSerializer {
+
+    static byte[] serialize(Owner owner) {
+        ByteArrayOutputStream bos = new ByteArrayOutputStream();
+        try {
+            ObjectOutputStream out = new ObjectOutputStream(bos);
+            out.writeObject(owner);
+            out.flush();
+            return bos.toByteArray();
+        } catch (IOException ignored) {
+            return new byte[0];
+        } finally {
+            try {
+                bos.close();
+            } catch (IOException ex) {
+                // ignore close exception
+            }
+        }
+    }
+
+    static Owner deserialize(byte[] bytes) {
```

Comment:
Can we add java docs on these methods? They should be private too once you add a `put()` and `get()` API.